liiatools/datasets/shared_functions/common.py

```diff
@@ -92,6 +93,32 @@ def save_year_error(input, la_log_dir):
         f.write(
             f"Could not process '{filename}' because no year was found in the name of the file"
         )
+        
+        
+def check_year_within_range(year, numofyears):
+    """
+    dummy docstring
```

Comment:
You should add a docstring to this so people understand how the function works, let me know if you need any help with this. You also need to take into account the current month, you can see an example of this in the remove_old_data() function in s903/lds_ssda903_la_agg/process.py. I would also probably make the month another parameter so this can be changed for different datasets that get submitted at different times in the year. And then make a unit test for this function